net.filebot.FileBot.yml

```diff
@@ -0,0 +1,86 @@
+app-id: net.filebot.FileBot
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: filebot.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --env=PATH=/usr/bin:/app/bin:/app/jre/bin
+  - --env=JAVA_HOME=/app/jre
+  - --share=network
+  - --filesystem=xdg-download
+  - --filesystem=xdg-videos
+  - --filesystem=xdg-music
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: zenity
+    buildsystem: meson
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/zenity.git
+        tag: 3.41.0
+  - name: chromaprint
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: archive
+        url: https://github.com/acoustid/chromaprint/archive/v1.5.0.tar.gz
+        sha256: 5c8e0d579cb3478900699110aa961c1552a422a18741cf67dd62136b1b877c7b
+  - name: libzen
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://mediaarea.net/download/source/libzen/0.4.39/libzen_0.4.39.tar.xz
+        sha256: cded5d7774294e7d5b42768ceb285243709b6e99eec6e980a3cf7f0890dbb622
+  - name: libmediainfo
+    subdir: Project/GNU/Library
+    config-opts:
+      - --enable-shared
+      - --disable-static
+      - --with-libcurl
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /lib/*.la
+    sources:
+      - type: archive
+        url: https://mediaarea.net/download/source/libmediainfo/21.09/libmediainfo_21.09.tar.xz
+        sha256: 69be9444f6a03a1bdf56713ee31d56e987c502a7ec18eb54ee70fcc171b7f126
+  - name: filebot
+    buildsystem: simple
```

Comment:
Just noticed the project license is set to `LicenseRef-proprietary`. Has upstream developer agreed to redistribute their app through our CDN?

Reply:
As linked in the initial post https://www.filebot.net/forums/viewtopic.php?p=56565#p56565

Not sure that's explicit enough?
He's also aware, that I'm doing this and asked for changes in https://www.filebot.net/forums/viewtopic.php?p=57218#p57218

Reply:
Okay, in the worst case it will need to be switched over to extra-data.